src/main/java/teammates/common/datatransfer/questions/FeedbackNumericalScaleQuestionDetails.java

```diff
@@ -607,15 +602,13 @@ private void populateSummaryStatisticsFromResponses(
             }
 
             // Compute average score received
-            double averageReceived = total.get(recipientEmail) / numResponses.get(recipientEmail);
-            average.put(recipientEmail, averageReceived);
+            average.put(recipientEmail, (total.get(recipientEmail) / numResponses.get(recipientEmail)));
 
             // Compute average score received excluding self
             averageExcludingSelf.putIfAbsent(recipientEmail, null);
             if (!isSelfResponse && totalExcludingSelf.get(recipientEmail) != null) {
-                double averageReceivedExcludingSelf =
-                        totalExcludingSelf.get(recipientEmail) / numResponsesExcludingSelf.get(recipientEmail);
-                averageExcludingSelf.put(recipientEmail, averageReceivedExcludingSelf);
+                averageExcludingSelf.put(recipientEmail, (totalExcludingSelf.get(recipientEmail) / 
```

Comment:
This line does not entirely coincide with the linting rules. Put the entire expression on a new line instead of just the denominator.